packages/animations/browser/src/render/shared.ts

```diff
@@ -140,51 +122,17 @@ export function parseTimelineCommand(command: string): [string, string] {
   return [id, action];
 }
 
-let _contains: (elm1: any, elm2: any) => boolean = (elm1: any, elm2: any) => false;
-let _query: (element: any, selector: string, multi: boolean) => any[] =
-    (element: any, selector: string, multi: boolean) => {
-      return [];
-    };
-let _documentElement: unknown|null = null;
+const documentElement: HTMLElement|null =
+    /* @__PURE__ */ (() => typeof document === 'undefined' ? null : document.documentElement)();
 
 export function getParentElement(element: any): unknown|null {
   const parent = element.parentNode || element.host;  // consider host to support shadow DOM
-  if (parent === _documentElement) {
+  if (parent == documentElement) {
```

Comment:
Is it important to have `==` here (vs `===`)?
```suggestion
  if (parent === documentElement) {
```